lib/helper.coffee

```diff
@@ -0,0 +1,91 @@
+ChildProcess = require 'child_process'
+Path = require 'path'
+FS = require 'fs'
+{XRegExp} = require 'xregexp'
+
+class Helpers
+  @findFile: (directory, fileName) ->
```

Comment:
Why attach all of these to the class? You could have constructed an object literal to return, or `module.exports = new Helpers` at the end of the file.